### ISSUE TITLE
GPU Support

### ISSUE THREAD

**Issue by BoPengGit** [puthurr/python-azure-function-gpu#1]
Hi,

So this method of using the Azure Functions will use the GPU from the AKS? Is this correct? 

Also how do we setup an API endpoint to send a request to this API?

Thank you and great work.

**Comment by puthurr** [comment]
That is correct. Today the OOTB Azure functions docker images don't support GPU hence this project. 

You need to apply the YAML onto your AKS cluster. You will find the provided sample under yaml/mrc-full-gpu.yaml. The deployed AKS service will give an endpoint to send queries to.  

Let me know if this helps.
